README.md

```diff
@@ -138,6 +138,7 @@ The following table lists the global parameters supported by the chart and their
 | Parameter | Description | Default |
 | -------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------- |
 | `global.enterprise` | Enable the enterprise features | `true` |
+| `global.auditlogs` | Enabled the auditlogs feature (requires enterprise) | `true` |
```

Comment:
I wonder gui uses `.Values.auditlogs.enabled`

Reply:
andso does workflows-worker

Reply:
You are right, doesn't make sense to have both flags.
I reworked the PR, let's use `auditlogs.enabled` everywhere.